include/arcade/IDisplay.hpp

```diff
@@ -117,7 +117,8 @@ namespace arcade
         /// @throws std::logic_error When @c font is not a font asset.
```

Comment:
Updating the font parameter's description to describe what `nullptr` does would be nice.